hawkular-alerters/hawkular-prometheus-alerter/deployments/hawkular-prometheus-alerter-standalone/src/main/webapp/WEB-INF/jboss-web.xml

```diff
@@ -18,5 +18,5 @@
 
 -->
 <jboss-web>
-  <context-root>/hawkular/__alerter_prometheus</context-root>
+  <context-root>/hawkular/alerter/prometheus</context-root>
```

Comment:
I unified the url with 2.x.
I guess there is no clash there and in this way, this endpoint are consistent with rest use cases.